### ISSUE TITLE
Stuck on "Updating Mint Properties" screen

### ISSUE THREAD

**Issue by gootconnect** [pkmnct/robinhood-mint-sync-chrome#43]
I run the initial setup and four accounts are created under the "property" category as expected, but the second Mint screen that opens up never closes and the values don't update. The screen stays on "Updating Mint Properties...". When I close out of everything and come back the four sub-accounts that were created for Robinhood are still there, but the values are $0.00. 

The extension was working as of a few days ago. It looks like this issue may have been looked at in the last update but I'm still having an issue.

**Comment by ethanwlo** [comment]
Seeing the same issue as @gootconnect described.

**Comment by superpowered** [comment]
Can you guys both turn on "Debug Mode" in the options and post what you see in the inspector console on the "Updating Mint Properties..." page?

**Comment by gootconnect** [comment]
I see 9 errors and 1 warning. I don't seem to be able to attach screen shots to the message board here. But I can save the text file of the errors and post that. It just looks terrible. 

I'm sorry I'm not more knowledgeable with code, I'm not sure what I'm looking at.

**Comment by superpowered** [comment]
Yeah, there's a bunch of errors that are just present by default on mint.. 

Mostly we just want to see the stuff from the debugger on the extension. It should all be prefixed with `Robinhood Mint Sync -`

**Comment by gootconnect** [comment]
Okay, here is what is coming up on the console when I filter out errors and warnings. The top three have the prefix you mentioned.

debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) initialized
chrome-extension://mogflmdandlpjobbddhopcggkjoggpdo/js/content/mint/triangleFix/triangleFix.js:1 Robinhood Mint Sync - [Content](Mint - Triangle Fix) initialized
chrome-extension://mogflmdandlpjobbddhopcggkjoggpdo/js/content/mint/triangleFix/triangleFix.js:1 Robinhood Mint Sync - [Content](Mint - Triangle Fix) Fixing triangle
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Using default sandbox factory...
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 All core plugins started...
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Plugins registered...
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Layout activated
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Layout activated
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Application before-ready completed...
libs-f0625f303abccdbf22c19e5c5570eeb9.js:4 Application ready...

**Comment by superpowered** [comment]
So just to verify:  there's no other logs after that?
The next log I would have expected to see is `Robinhood Mint Sync - [Content](Mint - Properties - Update) Property Tab View loaded. <div class=​"PropertyTabView">​…​</div>​
` 
Basically we wait to make sure that list of properties appears before we start to do anything (since thats where all the stuff we update shows up)

In the inspector you should be able to right click the console and get a "Save As" option. This will let you save the full log into a file to attach.

And also just to verify: on the Mint >Settings screen, behind the "Updating Mint Properties" overlay, do you see the various properties on your account? There should be at least the 4 created by the extension + any other properties you've added on your own.

**Comment by superpowered** [comment]
Also to verify: What version of the plugin do you show in your extensions?

**Comment by gootconnect** [comment]
I ran the whole process again and more Robinhood Mint Sync logs showed up. I filtered for logs containing "Robinhood". 

When I look on the Mint screen behind the "Updating Mint Properties" screen I see the other property account I added as well as the four created by the extension. 

Here are the logs I see after the last run. Still with the update screen not going away.

debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) initialized
chrome-extension://mogflmdandlpjobbddhopcggkjoggpdo/js/content/mint/triangleFix/triangleFix.js:1 Robinhood Mint Sync - [Content](Mint - Triangle Fix) initialized
chrome-extension://mogflmdandlpjobbddhopcggkjoggpdo/js/content/mint/triangleFix/triangleFix.js:1 Robinhood Mint Sync - [Content](Mint - Triangle Fix) Fixing triangle
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Waiting for Property Tab View to load
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Property Tab View loaded. <div class=​"PropertyTabView">​…​</div>​
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Attempting to set Cash to 0
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Expanding property Cash <li class=​"AccountView OtherPropertyView OtherPropertyAccountView open">​…​</li>​
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Found Cash input, setting amount <input value=​"$0.00" data-validate=​"{"amount":​{"val":​0, "messageError":​"Please enter a valid amount"}​}​" maxlength=​"17" autocomplete=​"off" placeholder=​"$" type=​"text" name=​"value" id=​"PFM:​OtherPropertyAccount:​15213245_12529055_value">​
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Attempting to set Crypto to 0
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Expanding property Crypto <li class=​"AccountView OtherPropertyView OtherPropertyAccountView open">​…​</li>​
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Found Crypto input, setting amount <input value=​"$0.00" data-validate=​"{"amount":​{"val":​0, "messageError":​"Please enter a valid amount"}​}​" maxlength=​"17" autocomplete=​"off" placeholder=​"$" type=​"text" name=​"value" id=​"PFM:​OtherPropertyAccount:​15213245_12529054_value">​
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Attempting to set Stocks to 0
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Expanding property Stocks <li class=​"AccountView OtherPropertyView OtherPropertyAccountView open">​…​</li>​
debug.ts:26 Robinhood Mint Sync - [Content](Mint - Properties - Update) Found Stocks input, setting amount <input value=​"$0.00" data-validate=​"{"amount":​{"val":​0, "messageError":​"Please enter a valid amount"}​}​" maxlength=​"17" autocomplete=​"off" placeholder=​"$" type=​"text" name=​"value" id=​"PFM:​OtherPropertyAccount:​15213245_12529056_value">​


The version I see is 3.3.0

I probably should have checked that first and updated to the latest...

**Comment by superpowered** [comment]
Well in asking that question I'm now realizing that 3.3.0 is the latest on the chrome store. 

3.4.0 addressed an issue with the "Other" category which seems to be where yours is hanging at based on that log.

My assumption is 3.4.0 has been submitted and must be waiting on approval from google. @pkmnct would be able to confirm that

**Comment by pkmnct** [comment]
Yes! This bug was found and resolved in 3.4.0. Unfortunately, there was an issue with the build script when 3.3.1 was submitted, resulting in a zip that was missing the compiled code getting uploaded to the Chrome Web Store. That has been stuck in "pending review" for a few days. There is no option to cancel and re-submit the corrected version until the broken one gets through review. See the messages below from Chrome Web Store support:

> I'd like to cancel review of this version. I realized there was an issue with my build script that caused the compiled code to not be included in the newly-uploaded .zip. I have a fix, but I am unable to submit it for review due to the current (broken) version pending review.

> Hello George,
> 
> Thanks for reaching out to us.
> 
> Unfortunately that is something we cannot do. Once a request is in the system, it has to go through the whole process and we don't have a mechanism to cancel a review. We'll bring this up with the team and see if this is something we can implement in the future.
> 
> Regards,
> 
> Oliver
> 
> Chrome Web Store Developer Support

Hopefully the review will happen soon, and I can get the fix published.

**Comment by superpowered** [comment]
@pkmnct could probably start including "Releases" in the github repo, so people can download targeted versions of the app when needed.

**Comment by pkmnct** [comment]
Agreed! I have a branch where I've started to implement the automated release pipeline.

**Comment by gootconnect** [comment]
Well it sounds like everything is good, I just need to wait a bit. Thanks for the responses!

**Comment by pkmnct** [comment]
At this time, the plugin is still pending review. [According to the Webstore Developer FAQs](https://developer.chrome.com/docs/webstore/faq/#faq-listing-108):
> Review times vary; some reviews complete in a few hours, others take many days, and in some cases a review can take several weeks.

I am hopeful this is just due to the broken build having been uploaded. I will post again after I am able to submit version 3.4.0 (or whatever is the latest).

**Comment by pkmnct** [comment]
The broken submission has been rejected. This change has been submitted to the Chrome Web Store and Firefox Add-on site.

**Comment by pkmnct** [comment]
The update was published. Please open a new issue if you continue running into this after you get the latest update.
